### DISCUSSION TITLE
Is there a simple way to display code block ?

### DISCUSSION THREAD

**Discussion by cathino** [xdan/jodit#1122] [Q&A] [answered]
I'd like to make an editor a bit like some markdown editor which can inculde code block. Is there a simple way to display or edit code block ?

**Comment by xdan** [comment, marked as answer]
Core Jodit doesn't have a dedicated code-block button — you can insert a `<pre>` and it'll keep it, but there's no syntax-highlighted block out of the box. Jodit PRO has a code/source-highlighting plugin if you need real highlighted blocks.

For a lightweight DIY in the free version, add a small custom button that wraps the selection in `<pre><code>…</code></pre>`. Want the snippet?
